draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1099,7 +1099,17 @@
 <t>
    Because unsafe request methods (<xref target="safe.methods"/>) such as PUT, POST or DELETE
    have the potential for changing state on the origin server, intervening
-   caches can use them to keep their contents up to date.
+   caches are required to invalidate stored responses to keep their contents up to date.
+   <x:dfn>Invalidate</x:dfn> means that the cache will either remove all
+   stored responses related to the effective request URI or will mark these
+   as "invalid" and in need of a mandatory validation before they can be sent
+   in response to a subsequent request.
+</t>
+<t>
+   Note that this does not guarantee that all appropriate responses are
+   invalidated globally. For example, a state-changing request might invalidate
+   responses in the caches it travels through, but relevant responses still
+   might be stored in other caches that it has not.
```

Comment:
I'm not sure if this needs to be said, but if it does then I think it reads better as:

```
   Note that this does not guarantee that all appropriate responses are
   invalidated globally: a state-changing request would only invalidate
   responses in caches that it travels through.
```